guides/common/modules/proc_installing-capsule-server-packages.adoc

```diff
@@ -16,17 +16,13 @@ ifdef::foreman-deb[]
 include::proc_installing-proxy-packages.adoc[]
 endif::[]
 
-ifdef::foreman-el,katello[]
-== [[centos-7-package]]{RHEL} 7 / CentOS Linux 7
-endif::[]
-
 ifdef::foreman-el,katello,satellite[]
+[[el-7-package]] == {EL} 7
```

Comment:
I now see I swapped the order. Should `==` be first or doesn't it matter?

Reply:
I am not 100% sure but think you're correct. This should "force" asciidoctor to add an anchor to "EL 7" named "el-7-package". Maybe you can check this in the build artifacts.

Reply:
I now swapped it around because that's what others do as well. Even if both work, this is more consistent which is a benefit in itself.